src/index.ts

```diff
@@ -29,52 +33,89 @@ export default {
 
     const supabase = createClient(env.SUPABASE_URL, env.SUPABASE_KEY);
```

Comment:
```suggestion
    const supabase = createClient(env.SUPABASE_URL, env.SUPABASE_ANON_KEY);
```

We changed the secrets org wide to indicate ANON_KEY